presto-main/src/main/java/io/prestosql/sql/planner/plan/JoinNode.java

```diff
@@ -143,6 +143,11 @@ public JoinNode(
         }
     }
 
+    public static boolean requiresEquiJoinClauses(DistributionType distributionType, Type type)
```

Comment:
add a comment why it's requried.

Reply:
let's continue https://github.com/prestosql/presto/pull/3228#discussion_r398030611

from JoinNode perspective, this is just a trivial factoring out of a method, without any change in the logic